lib/sharing/mutable_gist.dart

```diff
@@ -38,21 +38,20 @@ class MutableGist implements PropertyOwner {
 
   bool? get public => _backingGist.public;
 
-  MutableGistFile? getGistFile(String? name) {
-    if (_files[name] == null) {
-      _files[name] = MutableGistFile._(this, name);
+  MutableGistFile getGistFile(String name) {
+    var fileWithName = _files[name];
```

Comment:
Could this line and the following line be collapsed ionto one with a joining `??` operator?

Reply:
Good suggestion, thanks!